src/kirby/components/divider/divider.component.ts

```diff
@@ -1,8 +1,11 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 
 @Component({
   selector: 'kirby-divider',
   templateUrl: './divider.component.html',
   styleUrls: ['./divider.component.scss'],
 })
-export class DividerComponent {}
+export class DividerComponent {
+  @Input()
+  hasMargin: boolean;
```

Comment:
Shouldn't this be initialized to `true`, to be backwards compatible with existing usage of the component?

Reply:
As of @bommariusser's comment (https://github.com/kirbydesign/designsystem/issues/805#issuecomment-561622108), this should be `false` initially

Reply:
I have added a "BREAKING CHANGE" label